### DISCUSSION TITLE
Rating View plugin for MAUI

### DISCUSSION THREAD

**Discussion by Eel2000** [CommunityToolkit/Maui#1495] [New Feature Discussions]
Hi Team. I've developed a plugin for MAUI that allows adding Rating Capability into a Maui application and would like it to be a part of the community toolkit. here is an overview of how its work and the ref to the repo. The nuget package is still in preview but is ready to be used in any MAUI app

https://github.com/CommunityToolkit/Maui/assets/44249870/154dec78-3682-4b6d-abb8-d692f95f13e0

repo : https://github.com/Eel2000/Maui.RatingView

Nuget : https://www.nuget.org/packages/RatingViewControl/

**Comment by GeorgeLeithead** [comment]
Thanks Eliezer, this is very much appreciated.  I hope that we could get this raised at the next stand-up for discussion, to see if this would be of benefit to the comminity as a dedicated control. @brminnick @bijington

**Reply by bijington** [reply]
I think based on our discussion rules we are supposed to wait until it gets 10 votes. I have upvoted it though as I really like the idea

**Reply by Eel2000** [reply]
Alright. I'll be waiting then.

**Comment by CliffAgius** [comment]
I like this so added my vote. 😜

**Comment by hervino14** [comment]
I like it

**Comment by CliffAgius** [comment]
Hey @Eel2000 we discussed this on todays stand-up and wondered if you would like to join the maintainers team and open this as an issue to work on bringing into the MCT?

**Reply by Eel2000** [reply]
Hy @CliffAgius , I would love it. As I have too much to learn from you and the community. It will be a pleasure.

**Reply by CliffAgius** [reply]
paging @brminnick can you add our new maintainer to the team please. ;-p

**Reply by TheCodeTraveler** [reply]
Yup! We're doing the onboarding on Wednesday

**Comment by CliffAgius** [comment]
Closing this discussion as the feature is being worked on with an initial PR here #1706
